auto-ipsec/libreswan/README.md

```diff
@@ -4,33 +4,33 @@ Using the certificate distribution mechanism built into keylime, you can automat
 
 ## Overview
 
-These scripts allow keylime nodes to create a virtual private network between them that is protected by cryptographic keys bootstrapped by keylime.  Keylime will also automatically remove failed nodes from the network.  The scripts work by leveraging two features of keylime: automatic certificate generation/delivery, and the ability run scripts provided in the payload after successful bootstrapping.  The following high level actions will happen:
+These scripts allow keylime agents to create a virtual private network between them that is protected by cryptographic keys bootstrapped by keylime.  Keylime will also automatically remove failed agents from the network.  The scripts work by leveraging two features of keylime: automatic certificate generation/delivery, and the ability run scripts provided in the payload after successful bootstrapping.  The following high level actions will happen:
```

Comment:
this paragraph and the next one, we're not really talking about the agent, but the machine that keylime is running on.  we're not really making it so that agents are talking over ipsec.  suggest replace with machine? or leave node?

Reply:
I agree, there should be a distinction between the machine/node/system and the Keylime agent (software) running on it.

Reply:
I used host for (sounds a little better than machine)

Reply:
Has this already been resolved?  For instance, the below paragraph says, "the tenant must specify the subnets/ips that they wish for **agents** to always use IPsec to communicate."  

This makes it sound like the Keylime agent software is communicating over IPsec, but it will not be.  These scripts will set up a network interface for IPsec that nodes can use to communicate directly with each other (if they want to use it), but Keylime agents will not use this to communicate with the verifier/tenant. The same issue exists in `racoon/README.md`.  

We need to make it more clear when we're talking about the cloud machine and when we're talking about the Keylime agent software.

Reply:
Understood, please take a look at the last commit 613fda559347ced0e7a6f51aa2659abd500bb707

This should address the naming.

Reply:
Yep, that looks good @lukehinds !